app/controllers/quotes_controller.rb

```diff
@@ -2,7 +2,7 @@ class QuotesController < ApplicationController
   before_action :set_quote, only: %i[show edit update destroy]
 
   def index
-    @quotes = Quote.all
+    @quotes = Quote.ordered
```

Comment:
`ordered` is a `scope` that orders the records in a certain way. This line ensures the controller passes them to the view according to that specification.